webapp/pages/profile/_id/_slug.vue

```diff
@@ -417,6 +421,24 @@ export default {
         this.$apollo.queries.profilePagePosts.refetch()
       }
     },
+    async blockUser(user) {
+      try {
+        await this.$apollo.mutate({ mutation: blockUser(), variables: { id: user.id } })
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T17:32:46Z" title="Thursday, January 23rd 2020, 6:32:46 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -417,6 +420,24 @@ export default {
         this.$apollo.queries.profilePagePosts.refetch()
       }
     },
+    async blockUser(user) {
+      try {
+        await this.$apollo.mutate({ mutation: blockUser(), variables: { id: user.id } })
```


```suggestion
        await this.$apollo.mutate({ mutation: blockUser(), variables: user })
```
Would be OK, too, I guess. I'm OK to only pass the `id` call though. Choose whatever you like more. Maybe the explicit `id` is more, well, explicit.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:05:38Z" title="Thursday, January 23rd 2020, 7:05:38 pm +01:00">Jan 23, 2020</time>_

---


I copied what you added when you introduced the block feature... and I guess it's good enough ;)